app/models/infrastructure_facades/pl_grid_facade.rb

```diff
@@ -70,8 +70,16 @@ def start_simulation_managers(user_id, instances_count, experiment_id, additiona
 
     if additional_params[:onsite_monitoring]
       sm_uuid = SecureRandom.uuid
-      PlGridFacade.send_and_launch_onsite_monitoring(credentials, sm_uuid, user_id,
+      begin
```

Comment:
Nie podoba mi się że masz ten sam kod w dwóch miejscach z dokładnością co do (PlGrid/PrivateMachine)Facade, potencjalnie kiedyś może się to rozjechać, jeśli się da proponowałbym zrobić z tego metodę w np InfastructureFacade.